credoai/lens.py

```diff
@@ -599,10 +600,13 @@ def _get_names(self):
     def _init_assessments(self):
         """Initializes modules in each assessment"""
         for assessment in self.assessments.values():
-            kwargs = self.spec.get(assessment.name, {})
-            assessment.init_module(model=self.model,
-                                   data=self.data,
-                                   **kwargs)
+            kwargs = deepcopy(self.spec.get(assessment.name, {}))
```

Comment:
@amrasekh , this is how I am using the requirements of an assessment to determine if it requires "model" or "data".  I can imagine us doing soemthing similar when deciding whether to export to "model" or "data"